### DISCUSSION TITLE
HASPmota no longer sends hold-event

### DISCUSSION THREAD

**Discussion by waltkind** [arendst/Tasmota#22408] [Q&A] [answered]
I have upgraded my Lanbon L8 from 14.1.0 to 14.3.01 recently.
Some of my scripts no longer worked as expected as I checked for the hold-event.
When I touch the screen in 14.1.0 the output is 'down - long - hold - release - up' .
In 14.3.0.1 it is 'down - long - release -up'.

When was this behavior been changed? I have seen nothing in the changelog.

**Comment by smcgann99** [comment, marked as answer]
It was recently changed here - https://github.com/arendst/Tasmota/pull/22194.
I was doing some testing and reported a problem with change events not working, the above patch was to address that.
While it fixed the change events then, they now seem to have stopped reporting the text and val attributes again since then.
Out of interest could I ask, does your script need repeated hold events or would 1 suffice to trigger it?
e.g. down - long - hold - release -up

**Reply by waltkind** [reply]
I changed the scripts from 'hold' to 'long' and everything is OK.
So for me now it is fine. I only had to find the issue and in the changelog I found nothing that would concern me.
